types/index.d.ts

```diff
@@ -212,6 +212,10 @@ declare class NodeCanvasRenderingContext2D extends CanvasRenderingContext2D {
 	createPattern(...args: any[]): NodeCanvasCanvasPattern
 	createLinearGradient(x0: number, y0: number, x1: number, y1: number): NodeCanvasCanvasGradient;
 	createRadialGradient(x0: number, y0: number, r0: number, x1: number, y1: number, r1: number): NodeCanvasCanvasGradient;
+	/**
+	 * An additional method .addPage() is then available to create multiple page PDFs
+	 */
+	addPage(width?: number, height?: number): void
```

Comment:
Oops, good catch. Can you delete the incorrect declaration from line 130-134 (I had it on the Canvas instance instead of the rendering context) please? You could also copy the text from it that describes the default width/height values.

Reply:
Updated!